### PR TITLE
ToC reorganization, part 2

### PR DESCRIPTION
**Pull request by melissawm**
This is a second part of the ToC reorganization and should be merged right after napari/napari#3751.

### CODE REVIEW

**Comment by melissawm** [comment]
@neuromusic @GenevieveBuckley this concludes the largest part of the reorg, and should be merged soon to get users access to all documents from the ToC in the web page - let me know if there are any concerns and I'll be happy to fix right away. Thanks!

File: _toc.yml [hunk 1]
```diff
@@ -1,45 +1,80 @@
...
+    - file: plugins/stable/find-and-install-plugin
+    - file: tutorials/fundamentals/viewer
+    - file: tutorials/fundamentals/image
+    - file: tutorials/fundamentals/labels
+    - file: tutorials/fundamentals/points
+    - file: tutorials/fundamentals/shapes
+    - file: tutorials/fundamentals/surface
+    - file: tutorials/fundamentals/tracks
+    - file: tutorials/fundamentals/vectors
+    title: Getting Started
```
Comment (melissawm) [inline _toc.yml:16]: I know we had discussed putting the title in the beginning of the section, but because this is a yaml dump I thought it could be either ignored or left to another PR.

File: tutorials/applications/napari_imageJ.md [hunk 2]
```diff
@@ -1,148 +0,0 @@
-# napari + ImageJ how-to guide
```
Comment (melissawm) [inline tutorials/applications/napari_imageJ.md:1]: This document now lives in the how-to guides, so it was moved in the PR to napari/napari.

**Review by neuromusic — APPROVED** [review]

**Comment by neuromusic** [comment]
fantastic @melissawm !!

**Review by sofroniewn — APPROVED** [review]
This looks great!! I'm approving. I will give the australia timezone folks have a chance to review and then if all looks good tomorrow morning i will merge, or if they review tonight and want to merge away that's great too. thanks all!!

File: _toc.yml (hunk 1 above)
Comment (GenevieveBuckley) [reply _toc.yml:16]: Ok, no worries

**Comment by GenevieveBuckley** [comment]
Thank you @melissawm for pushing this forwards!

I've merged the changes, please do let us know if you notice anything odd about the display on the website (sounds like the napari-bot problems were fixed, just want to be sure nothing else weird happens)

**Comment by melissawm** [comment]
Everything looks good so far on my side, I'll keep an eye out for any remaining issues. Thanks, folks!
